include/NovelRT.Interop/Input/NrtNovelKey.h

```diff
@@ -0,0 +1,28 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+#ifndef NOVELRT_INTEROP_INPUT_NRTNOVELKEY_H
+#define NOVELRT_INTEROP_INPUT_NRTNOVELKEY_H
+
+#include "../NrtTypedefs.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    NrtNovelKeyInfo Nrt_NovelKey_ReadAsInfo(NrtNovelKeyHandle key);
+
+    const char* Nrt_NovelKey_GetKeyName(NrtNovelKeyHandle key);
```

Comment:
@RubyNova just to confirm, should we be directly outputting a `const char*` here? Or should we be returning a `NrtResult` and using a `const char**` parameter? We seem to use output parameters in other places but its not consistent.

Reply:
We originally agreed for getter only noexcept methods (so, pure property getters) we'd just do this - but that was a long time ago. You, as code owner, might feel this is no longer a good idea; but I think this is how it works.